starter-examples/starter-ranking.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
<p>Why do we need 4 moc and 4 column_specs? Could we add these in together?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/truera-examples/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Refactored things to remove the mocs and to use fewer column_specs.